README.md

```diff
@@ -1,37 +1,52 @@
-## Welcome to GitHub Pages
+# The Grand Auction (Capstone 1)
 
-You can use the [editor on GitHub](https://github.com/iBel251/grand-auction/edit/main/README.md) to maintain and preview the content for your website in Markdown files.
+## Description
+**This Project about The Grand Auction 2022 contains the program, featured items, and sponser sections**
 
-Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
+  **My activities on this project**
 
-### Markdown
+  1. Created project files using githup work flow.
+  2. Create two pages Home and about.
+  3. Add mobile menu and Add items section dynamically using JS.
+  4. Include auction events.
+  5. stick to design, colors, and fonts in project requirments.
+  
+  **All images and names used in this website are for learning and display purposes only.**
 
-Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for
+## Built With
 
-```markdown
-Syntax highlighted code block
+- Major languages:- Html, Css and Javascript
 
-# Header 1
-## Header 2
-### Header 3
+- Frameworks:- VS code
 
```

Comment:
- [ ] **Required** It is nice to have a `Getting started` section with necessary details. Refer [here](https://github.com/microverseinc/readme-template/blob/master/README.md)
- Prerequisites
- Setup
- Install